person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative './nameable/nameable_decorator'
 
 class Person < Nameable
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
```

Comment:
- [ ] in the Person class A method for adding rentals should be present. Please see this Link to the [example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)